pom.xml

```diff
@@ -210,30 +204,16 @@
           <artifactId>maven-gpg-plugin</artifactId>
           <version>1.6</version>
         </plugin>
-        <plugin>
-          <groupId>org.jacoco</groupId>
-          <artifactId>jacoco-maven-plugin</artifactId>
-          <version>0.7.5.201505241946</version>
-        </plugin>
       </plugins>
```

Comment:
@armfergom the maven plugins' versions should be kept managed. The last version of JaCoCO, for example, is not working properly with SonarQube.